tests/test_questions.py

```diff
@@ -934,23 +934,18 @@ def question(self, **kwargs):
     def test_value(self):
         question = self.question().summary({'example': ['val1.1', 'val2.2', 'val4']})
         assert question.value == [
-                {
-                    "label": "Option 1",
-                    "value": "val1",
-                    "options": [
-                        {"label": "Option 1.1", "value": "val1.1", "options": []},
-                    ]
-                },
-                {
-                    "label": "Option 2",
-                    "value": "val2",
-                    "options": [
-                        {"label": "Option 2.2", "value": "val2.2", "options": []},
-                    ]
-                },
-
-                {"label": "Option 4", "value": "val4", "options": []},
-            ]
+            {
+                "label": "Option 1", "value": "val1", "options": [
```

Comment:
I think this was easier to follow with each key on its own line.

Reply:
Fixed, best of both worlds 👍